frontend/src/concepts/k8s/pods/utils.ts

```diff
@@ -14,32 +14,41 @@ export const downloadFullPodLog = async (
   namespace: string,
   podName: string,
   containerName: string,
+  podCompleted: boolean | undefined,
 ) =>
   getPodContainerLogText(namespace, podName, containerName).then((content) =>
-    downloadString(`${podName}-${containerName}.log`, content),
+    downloadString(
+      `${podName}-${containerName}-${podCompleted ? 'completed' : 'incomplete'}.log`,
```

Comment:
I think this will be`complete` instead of `completed`. @yih-wang @kaedward For a single step, the name of the downloaded log file is `podname-stepname-complete` or `podname-stepname-incomplete`.

Reply:
Same here: https://github.com/opendatahub-io/odh-dashboard/pull/2159#discussion_r1396254019